neqo-transport/src/packet.rs

```diff
@@ -855,16 +875,23 @@ mod tests {
     }
 
     const SAMPLE_RETRY: &[u8] = &[
```

Comment:
As long as we support multiple versions, we will want to check that we can handle multiple Retry packets, so this should add, not modify the vector.

Reply:
ok, took a stab at resolving this, lmkwyt